codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/endpoints/discovery/EndpointDiscoveryIntegration.kt

```diff
@@ -22,26 +22,38 @@ import software.amazon.smithy.model.shapes.OperationShape
 import software.amazon.smithy.model.shapes.ServiceShape
 
 class EndpointDiscoveryIntegration : KotlinIntegration {
-    override fun additionalServiceConfigProps(ctx: CodegenContext): List<ConfigProperty> {
-        val endpointDiscoveryOptional = ctx
+    companion object {
+        const val CLIENT_CONFIG_NAME = "endpointDiscoverer"
+        const val ORDER: Byte = 0 // doesn't depend on any other integrations
```

Comment:
The default value is zero, so can this be removed?

Reply:
I now see this is used in aws-sdk-kotlin: https://github.com/awslabs/aws-sdk-kotlin/pull/1506/files#diff-707de3299374cf5e83fa4c28d54258c5e4f90cd575efa9da15580d454657d953R25